test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperDescriptorProviderTest.cs

```diff
@@ -10,7 +10,78 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
     public class TagHelperDescriptorProviderTest
     {
         [Fact]
-        public void TagHelperDescriptorProvider_GetTagHelpersReturnsNothingForUnregisteredTags()
+        public void GetTagHelpers_OnlyUnderstandsSinglePrefix()
```

Comment:
test what happens when a catch-all descriptor has prefix "th" and `// Act` calls `provider.GetTagHelpers("th")`.  suspect for example `<th/>` will lead to an odd error or generation of a self-closing tag with no name.